### ISSUE TITLE
Outdated dependencies

### ISSUE THREAD

**Issue by no-todos** [LouisBrunner/dnd-multi-backend#67]
### What is the problem
Currently `dnd-multi-backend` relies on outdated versions of dependencies

### Additional info
```
warning " > react-dnd-multi-backend@6.0.2" has incorrect peer dependency "react@^16.13".
warning " > react-dnd-multi-backend@6.0.2" has incorrect peer dependency "react-dnd-html5-backend@^11.1.3".
warning " > react-dnd-multi-backend@6.0.2" has incorrect peer dependency "react-dnd-touch-backend@^11.1.3".
warning " > react-dnd-multi-backend@6.0.2" has incorrect peer dependency "react-dom@^16.13".
```

### Solution
It'd be nice to see updated peer dependencies to include `react@^17`, `react-dom@^17`, `react-dnd-html5-backend@^12` and `react-dnd-touch-backend@^12`

**Comment by tomothumb** [comment]
same issue

**Comment by alarinel** [comment]
same

**Comment by LouisBrunner** [comment]
Hi,

I will be publishing `7.0.0-alpha.2` today, which will include compatibility with `react-dnd@12` and `react@17`.
If you can try that version and get back to me, I will get it published as `7.0.0`.
